CHANGELOG.md

```diff
@@ -139,6 +139,8 @@ The APIs of these subpackages are now stable, and all others are experimental. S
 * The speed of `NucleotideSequence.reverse_complement` has been improved (~6x).
 
 ### Bug fixes
+* Changed `skbio.stats.composition.multiplicative_replacement` to raise an
```

Comment:
This is in the wrong bug fix section (version 0.4.0). It needs to go under a new bug fix section for version 0.4.1-dev. @mortonjt or @ebolyen can you please submit a PR to fix this?